packages/circuit-ui/components/TopNavigation/TopNavigation.tsx

```diff
@@ -24,14 +24,22 @@ import Hamburger, { HamburgerProps } from '../Hamburger';
 import { ProfileMenu, ProfileMenuProps } from './components/ProfileMenu';
 import { UtilityLinks, UtilityLinksProps } from './components/UtilityLinks';
 
+export const TOP_NAVIGATION_HEIGHT = '49px'; /* height - border-bottom */
```

Comment:
Isn't this `height + border-bottom`? (this was the previous comment)

Reply:
Yes, I changed this and forgot to update the comment.